src/main/java/com/iexec/sms/admin/AdminController.java

```diff
@@ -56,6 +69,22 @@ public ResponseEntity<String> createBackup() {
         }
     }
 
+    /**
+     * Endpoint to restore a database backup.
+     * <p>
+     * This method allows the client to initiate the restoration of a database backup
+     * from a specified dump file, identified by the 'fileName', locate in a location specified by the 'storageID'.
```

Comment:
For your information, JavaDoc doesn't understand back quotes (`` ` ``). We can use `@literal` instead.

Reply:
I agree on the interest of `@literal` here. To be fair, back quotes were not used here, only simple quotes.

Reply:
using html <b></b> seems to be a better option, agree ?